src/main/java/org/time2java/tRussianBanr/domain/Status.java

```diff
@@ -4,4 +4,10 @@
  * Created by time2die on 08.11.16.
  */
 public class Status {
+    Integer bankCapital ;
+    Integer debt ;
```

Comment:
\[Checkstyle\] ERROR: Variable 'debt' must be private and have accessor methods\.